docs/software/apps-and-envs/gaussian.md

```diff
@@ -142,6 +142,7 @@ An example of the job script to use GPU is given below.
 #SBATCH --cpus-per-task=8
 #SBATCH --mem=32GB            # replace this with the memory on the CPU side
 #SBATCH --gres=gpu:1
+#SBATCH --constraint=a100
 
 module load gaussian/16RevC.02
 
```

Comment:
Thanks for testing! Let me double check on my end.

Reply:
@od101 I found that `--exclusive` is needed to run on b3 and m3 gpu, because it seems to me that Gaussian needs to access to all the CPU cores on the node to index the cores that control the GPU(s). Otherwise, I got a nonsense error of "integer division by zero" for the same input that runs fine without the GPU. Good thing is that Rev.C.02 runs normally with A40 GPUs.